frontend/src/components/elements/Input.js

```diff
@@ -37,10 +37,11 @@ const StyledInput = styled.input`
 //   display: block;
 // `;
 
-const Input = ({ className, type, name, placeholder, onChange, required, disabled, index }) => {
+const Input = ({ label, className, type, name, placeholder, onChange, required, disabled, index }) => {
 
   return (
     <StyledLabel className={className}>
+    {label && label}
       <StyledInput
         required={required && "required"}
         disabled={disabled && "disabled"}
```

Comment:
Why not simply `disabled={disabled}` and `required={required}`
Why would you need the 'string' and isn't a boolean enough to check if this value is set or not?

Reply:
👯